### PR TITLE
Api failure unit test

### PR DESCRIPTION
**Pull request by ruwang**

### CODE REVIEW

File: src/flags/watchjs [hunk 1]
```diff
@@ -1 +1 @@
-1
+1
```
Comment (ruwang) [inline src/flags/watchjs]: I might consider the localstorage now.

Comment (cdaringe) [inline src/flags/watchjs]: cool :)

File: src/transferMeta.js [hunk 2]
```diff
@@ -28,33 +26,25 @@ module.exports = (options) => {
...
         } else { // eslint-disable-line
           const codeError = `'Error Code:', ${code}`;
 
           // Notify admin after 5 tries and place a hold flag for watchjs
           if (retryAttempt > 5 && !notified) {
             emailMsg.text = codeError;
-            sendMail(emailMsg, () => {
-              notified = true;
+            notified = true;
+            sendMail(emailMsg, (success) => {
```
Comment (cdaringe) [inline src/transferMeta.js]: race still exists, but we can deal later. we have it tracked

File: src/utils/email-notification.js [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-bbify.chirp();
 const request = Promise.promisifyAll(require('request'));
 const logger = require('../utils/globals.js').logger;
 const config = require('config');
 
 module.exports = (emailOpts, cb) => {
   // apiUrl should be replaced with url for api key
-  request.getAsync(config.get('apiUrl'))
-    .then((apiKey) => {
+  request.getAsync(config.get('sendgridKey'))
```
Comment (cdaringe) [inline src/utils/email-notification.js]: it doesn't matter yet, clearly, as `halfpenny` and other routes are not yet ready. however, we will probably want to just stub in a bunch of stuff in this file, s.t. we don't make network requests.  we will probably also want to btoa/atob the sendgrid key before persisting to disk (if storing the key is still what we want to do long term).  im still shaky on passing the key to the app--unsure if their are security implications.  will have to study the sendgrid API, and make sure no private COINS creds or naughty operations can be accessed with that API key.  hopefully it's not, and it's pretty just a token to allow sending messages!  i'd hate for a "bad guy" to get our key, then send naughty emails to our users disguised as us using our sendgrid key.  i know i've been back and forth on this, so sorry about that.  it's a complicated thing!

Comment (ruwang) [inline src/utils/email-notification.js]: Good thought. we will need to revisit this at some time.

**Pull request merged**